kedro/framework/cli/micropkg.py

```diff
@@ -246,7 +246,7 @@ def _pull_package(  # noqa: PLR0913
         package_name = packages[0]
 
         # Type ignored because of https://github.com/pypa/build/pull/693
-        package_reqs = _get_all_library_reqs(library_meta)  # type: ignore[arg-type]
+        package_reqs = _get_all_library_reqs(library_meta)
```

Comment:
Is this change still needed?

Reply:
Good catch, not anymore after Ankita changes